### PR TITLE
Bugfix: Circle emitter injection

### PR DESCRIPTION
**Pull request by KZhu-ME**
Fixed the bug where the circle emitter injected ~25% of the particles inside the assembly object, compared to warp's 0%. Now it injects 0% inside the object.

### CODE REVIEW

File: mewarpx/mewarpx/emission.py [hunk 1]
```diff
@@ -1140,7 +1143,8 @@ def __init__(self, conductor, T, res_fac=5., transverse_fac=1.0, **kwargs):
             oshape)
 
         self.contours = np.squeeze(skimage.measure.find_contours(
```
Comment (peterscherpelz) [inline mewarpx/mewarpx/emission.py:1148]: Add comment here for why 0.17 was chosen.

File: Source/FieldSolver/ElectrostaticSolver.cpp [hunk 2]
```diff
@@ -164,7 +164,7 @@ WarpX::AddSpaceChargeField (WarpXParticleContainer& pc)
     computePhi( rho, phi, beta, pc.self_fields_required_precision, pc.self_fields_max_iters, pc.self_fields_verbosity );
 
     // Compute the corresponding electric and magnetic field, from the potential phi
-    computeE( Efield_fp, phi, beta );
+    // computeE( Efield_fp, phi, beta );
```
Comment (peterscherpelz) [inline Source/FieldSolver/ElectrostaticSolver.cpp]: This doesn't look correct? I guess we just need @roelof-groenewald 's electrostatic refactor / gradient calculation, but let's make sure this doesn't introduce an additional change or reversion.

**Review by peterscherpelz — COMMENTED** [review]
Minor comments. I think if ElectrostaticSolver.cpp matches `memaster` and the other comment is addressed this can be merged.

File: Source/FieldSolver/ElectrostaticSolver.cpp (hunk 2 above)
Comment (KZhu-ME) [reply Source/FieldSolver/ElectrostaticSolver.cpp]: Thanks for pointing this out, I'm not sure how this happened.

**Review by peterscherpelz — APPROVED** [review]

File: mewarpx/tests/test_emission.py [hunk 3]
```diff
@@ -149,9 +150,14 @@ def test_circle_emitter():
         df[label + '_std'] = np.std(res_dict[label])
 
     assert testing_util.test_df_vs_ref(
-        testname="circle_emitter", df=df, margin=0.4
+        testname="circle_emitter", df=df, margin=0.3
     )
 
+    emitter.plot_contours()
+    plotfile =  f"{cylinder.name}_contour_plot.png"
```
Comment (EtherealMC-Bit) [inline mewarpx/tests/test_emission.py:157]: Do we save plots as PNGs or JPEGs, or does it even matter? PNG's are higher quality but I feel like the difference doesn't matter for these plots.

**Review by EtherealMC-Bit — APPROVED** [review]
Nice! Just one question

File: mewarpx/tests/test_emission.py (hunk 3 above)
Comment (roelof-groenewald) [reply mewarpx/tests/test_emission.py:157]: @MKieburtz  I don't think it matters a ton but lets stick with PNG's for the sake of consistency.

Comment (KZhu-ME) [reply mewarpx/tests/test_emission.py:157]: Good question, I'm not sure if there is a standard. I just went with PNG since I saw other examples using PNG.

**Pull request merged**